docs/beam_handling.rst

```diff
@@ -53,19 +53,122 @@ Multi-beam cubes
 ----------------
 
 Varying resolution (multi-beam) cubes are somewhat trickier to work with in
-general, though unit conversion is easy.  You can perform the same sort of unit
-conversion with `~spectral_cube.spectral_cube.VaryingResolutionSpectralCube` s
-as with regular `~spectral_cube.spectral_cube.SpectralCube` s; ``spectral-cube``
-will use a different beam and frequency for each plane.
+general, though some operations are similar to a single resolution cube.
+Unit conversion is one case where the operation is called the same for
+`~spectral_cube.spectral_cube.VaryingResolutionSpectralCube` s
+as with regular `~spectral_cube.spectral_cube.SpectralCube` s. For example, to
+convert from Jy / beam to K::
 
-You can identify channels with bad beams (i.e., beams that differ from a reference beam,
-which by default is the median beam) using
+    >>> vrsc_cube_K = vrsc_cube.to(u.K)  # doctest: +SKIP
+
+``spectral-cube`` will use a different beam and frequency for each plane.
+
+Handling variations in the beams is often a source of difficulty. Some spectral
+operations (e.g., moment maps) require a common resolution. However, a few channels
+may have large discprepancies in the beam compared to most of the others (e.g., if
+more data in a particular channel was bad and had to be flagged).
+`~spectral_cube.spectral_cube.VaryingResolutionSpectralCube` deals with these problems in two ways.
+
+Finding and convolving to a common resolution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When applying spectral operations on a
+`~spectral_cube.spectral_cube.VaryingResolutionSpectralCube`, it is often
+necessary to first convolve the data to have a common beam. To retain the
+maximum amount of spatial information, the data can be convolved to the
+smallest common beam that all beams in
+`~spectral_cube.spectral_cube.VaryingResolutionSpectralCube.beam` can be
+deconvolved from.
```

Comment:
this is... pretty hard to parse.  Try to rephrase?  Maybe just end at "smallest common beam"?  Isn't it "smallest beam that all beams can be convolved to" anyway?

Reply:
Fixed.